### PR TITLE
Set remoteHost/Addr/Port and localName/Addr/Port of Coyote request

### LINKED ISSUE #81: Remote address is not set in TomcatServiceInvocationHandler
It looks like at the moment remote address is not passed to coyote request. It's quite easy to do with some thing like this in TomcatServiceInvocationHandler.java (I modified the signature to pass the entire context):

```
    private static Request convertRequest(ServiceInvocationContext ctx) {

        final Request coyoteReq = new Request();
        InetSocketAddress remote = (InetSocketAddress)ctx.remoteAddress();
        //byte[] address = remote.getAddress().getAddress();

        coyoteReq.setRemotePort (remote.getPort());
        coyoteReq.remoteAddr().setString (remote.getHostName());

```

I had a servlet that was performing some authorization based on the remote address. With the change above is works fine now :)

### PR DESCRIPTION
**Pull request by trustin**
Related: #81

Motivation:

When TomcatServiceInvocationHandler converts a Netty HTTP request to a
Coyote request, it should set the remoteAddr() and remoteHost() property
of the Coyote request.

Modifications:

- Fill remoteAddr/remoteHost/remotePort/localAddr/localName/localPort of
  Coyote request
- Add ServiceInvocationContext.localAddress() to get the local address
  and port
- Fix the content-type of the test JSP pages

Result:

Fixes #81

### CODE REVIEW

**Comment by trustin** [comment]
@yfinkelstein Could you please confirm this PR fixes your problem?

**Comment by anuraaga** [comment]
Hmm thought I had replied to this but don't see my comment, let me try again - just wondering can we add unit tests for this or too hard to mock?

**Comment by trustin** [comment]
@anuraaga I can write a test case for this using my JSP-fu for sure. Stay tuned.

**Comment by trustin** [comment]
@anuraaga Updated.
